src/plugins/walletConnectToDapps/v2/components/modals/EIP155TransactionConfirmation.tsx

```diff
@@ -33,7 +33,8 @@ export const EIP155TransactionConfirmation: FC<
   WalletConnectRequestModalProps<EthSendTransactionCallRequest | EthSignTransactionCallRequest>
 > = ({ onConfirm: handleConfirm, onReject: handleReject, state }) => {
   const { address, transaction, isInteractingWithContract, method } = useWalletConnectState(state)
-  assertIsTransactionParams(transaction)
+
```

Comment:
We may or may not want to revert this, simply added to avoid renders where this would potentially be undefined